modules/akka-persistence-runtime/src/main/scala/aecor/util/PersistentEncoderCirceUtil.scala

```diff
@@ -1,4 +1,4 @@
-package aecor.example
+package aecor.util
```

Comment:
I don't want to introduce unneeded dependency on Circe in `akka-persistence-runtime` and also encourage JSON persistent encoders. This should be moved out along with example.

Reply:
Oki, will do.